src/examples/PopupExamplePage.tsx

```diff
@@ -14,7 +14,17 @@ export const PopupExamplePage: React.FunctionComponent<{}> = () => {
   const [showPopup3, setShowPopup3] = useState(false);
   const [showPopup4, setShowPopup4] = useState(false);
 
-  const myRef = useRef();
+  const popupRef : React.RefObject<Popup> = React.createRef<Popup>();
+  const viewRef: React.RefObject<View> = React.createRef<View>();
+
+  useEffect(() => {
+    if (popupRef.current?.props.isOpen) {
+      const reactTag = findNodeHandle(viewRef.current);
+      if (reactTag) {
+        AccessibilityInfo.setAccessibilityFocus(reactTag);
+      }
+    }
+  })
```

Comment:
We'll want to apply this setAccessibilityFocus code to all Popups on the page. So this logic should handle checks to see if those popups are open as well.

Reply:
I'll push the code change that applies the `setAccessibilityFocus` code to all the popup examples.